src/components/Home/Home.tsx

```diff
@@ -0,0 +1 @@
+export const Home = () => <h1 className="title">Home Page</h1>;
```

Comment:
```suggestion
export const HomePage = () => <h1 className="title">Home Page</h1>;
```

And it should be in `pages` folder